Update Add-RoleGroupMember.md (#10242)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/Add-RoleGroupMember.md

- Mail-enabled security groups (don't use in Security & Compliance PowerShell)
- Security groups (on-premises Exchange only)

> Kindly note that the Security Groups/Mail-enabled security groups cannot be added as a member of the role group in the Microsoft Purview Portal, please add them via PowerShell with this parameter.
> 

Reviewer: ```suggestion
Note that you cannot add security groups and mail-enabled security groups as members of the role group in the Microsoft Purview portal. You should add them via PowerShell with this parameter.
```
